### DISCUSSION TITLE
Очень хорошая программа

### DISCUSSION THREAD

**Discussion by Dumitru88888** [farag2/Sophia-Script-for-Windows#581] [Q&A] [answered]
Очень хорошая программа и бесплатная. Я бы хотел задать создателям такой вопрос?
Нужно добавить по больше функций для отключения ненужных объектов из виндовс. Вырезать всё чтобы виндовс была по легче. Облегчить оболочку.

**Comment by farag2** [comment, marked as answer]
Из Windows невозможно что-либо "вырезать". И "легче" ее сделать нельзя.
